lib/system/database.rb

```diff
@@ -252,7 +252,7 @@ def drop
              EXECUTE IMMEDIATE 'DROP PROCEDURE #{name}';
           EXCEPTION
             WHEN OTHERS THEN
-              IF SQLCODE != -4080 THEN
+              IF SQLCODE != -4043 THEN
```

Comment:
Yes it is that one easy :)

➡️ `ORA-04043: object SP_INVOICES_FRIENDLY_ID does not exist` 

➡️  **ORA-04043**
➡️ 4043